maintainers/maintainer-list.nix

```diff
@@ -6958,6 +6958,12 @@
     githubId = 36407913;
     name = "Uli Baum";
   };
+  xfix = {
```

Comment:
Can you please move this change into a separate commit, please?

Reply:
Done.

Reply:
It still looks like one commit, can you double check?

Reply:
That's because adding myself as a maintainer is already on master